docs/hub/post_install.md

```diff
@@ -27,145 +26,13 @@ lastupdated: "2021-02-20"
-   ibm-edge-exchangedb-proxy-6bbd5b485-cx2v8      1/1     Running     0          88m
-   ibm-edge-exchangedb-proxy-6bbd5b485-hs27d      1/1     Running     0          88m
-   ibm-edge-exchangedb-proxy-6bbd5b485-htldr      1/1     Running     0          88m
-   ibm-edge-exchangedb-sentinel-6d685bf96-hz59z   1/1     Running     1          88m
-   ibm-edge-exchangedb-sentinel-6d685bf96-m4bdh   1/1     Running     0          88m
-   ibm-edge-exchangedb-sentinel-6d685bf96-mxv2b   1/1     Running     1          88m
-   ibm-edge-sdo-0                                 1/1     Running     0          81m
-   ibm-edge-ui-545d694f6c-4rnrf                   1/1     Running     0          81m
-   ibm-edge-ui-545d694f6c-97ptz                   1/1     Running     0          81m
-   ibm-edge-ui-545d694f6c-f7bf6                   1/1     Running     0          81m
-   ```
-   {: codeblock}
-
-   Notes:
-   * For more information about any pods in the **Pending** state due to resource or scheduling issues, see the [cluster sizing](cluster_sizing.md) page. This includes information about how to reduce scheduling costs of components.
-   * For more information about any other errors, see [troubleshooting](../admin/troubleshooting.md).
-3. Ensure that all pods in the **ibm-common-services** namespace are either **Running** or **Completed**:
-
-   ```
-   oc get pods -n ibm-common-services
-   ```
-   {: codeblock}
-
-4. [Download the agent package from IBM Passport Advantage](part_numbers.md) to your installation environment and unpack the installation media:
-    ```
-    tar -zxvf ibm-eam-{{site.data.keyword.semver}}-agent-x86_64.tar.gz && \
-    cd ibm-eam-{{site.data.keyword.semver}}-agent-x86_64/tools
-    ```
-    {: codeblock}
-5. Validate the installation state:
-    ```
-    ./service_healthcheck.sh
-    ```
-    {: codeblock}
-
-    See the following example output:
-    ```
-    $ ./service_healthcheck.sh
-    ==Running service verification tests for IBM Edge Application Manager==
-    SUCCESS: IBM Edge Application Manager Exchange API is operational
-    SUCCESS: IBM Edge Application Manager Cloud Sync Service is operational
-    SUCCESS: IBM Edge Application Manager Agbot database heartbeat is current
-    SUCCESS: IBM Edge Application Manager SDO API is operational
-    SUCCESS: IBM Edge Application Manager UI is properly requiring valid authentication
-    ==All expected services are up and running==
-    ```
-
-   * If there are **service_healthcheck.sh** command failures, if you experience issues running the commands below, or if there are issues during runtime, see [troubleshooting](../admin/troubleshooting.md).
 
 
 ## Post installation configuration
 {: #postconfig}
 
 The following process must run on a host that supports installation of the **hzn** CLI, which currently can be installed on a Debian/apt based Linux, amd64 Red Hat/rpm Linux, or macOS host. These steps use the same media downloaded from PPA in the Installation verification section.
```

Comment:
Should this point/link to some place?

These steps use the same media downloaded from PPA in the Installation verification section.